src/components/SearchResults.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import TableRow from "./TableRow";
+
+const SearchResults = props => {
+  return (
+    <table className="table">
+      <thead className="thead-dark">
+        <tr>
+          <th scope="col">ID</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Last Name</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room #</th>
+          <th scope="col">Check-in</th>
+          <th scope="col">Check-out</th>
+          <th scope="col">Duration</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map((customer, index) => {
+          return <TableRow key={index} list={customer} />;
```

Comment:
Do you think `list` is the best name for this prop? What does it represent? Could we use that as a name instead?

Reply:
I have replaced it with a good descriptive name, thanks again